test/descriptive-link-text-override-config.md

```diff
@@ -0,0 +1,17 @@
+# Descriptive link text override config
+
+[Go here](https://example.com/javascript/about) {MD059}
+
+[Learn more](https://example.com/javascript/about).
+
+[Click here](https://example.com/javascript/about).
+
+To learn more, go [here!](https://example.com/javascript/about).
+
+To learn more, go to this [link!](https://example.com/javascript/about).
+
+<!-- markdownlint-configure-file {
+  "descriptive-link-text": {
+    "banned_link_texts_override": ["go here"]
```

Comment:
I love that you've added this test. Please have it verify two banned texts for added safety.